clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -370,4 +370,7 @@ def warn_missing_symbol_graph_dir : Warning<
   "Missing symbol graph output directory, defaulting to working directory">,
   InGroup<ExtractAPIMisuse>;
 
+def err_ast_action_on_unparsable_source : Error<
```

Comment:
nit: Maybe a different name for this would be better? It’s not really ‘unparseable’, seeing as we can parse LLVM IR just fine; it’s just not C++.

Reply:
Agree, I've changed the name to `err_ast_action_on_llvm_ir`